src/environment.cc

```diff
@@ -78,12 +78,14 @@ constexpr const char kGceInstanceResourceType[] = "gce_instance";
 }
 
 Environment::Environment(const Configuration& config)
-    : config_(config), application_default_credentials_read_(false) {}
+    : config_(config),
+      gce_metadata_server_address_(kGceMetadataServerAddress),
```

Comment:
Let's just call it `metadata_server_url_`.

Reply:
Done.